consensus/src/validation/handler.rs

```diff
@@ -214,6 +214,14 @@ impl MsgHandler for ValidationHandler {
                     )
                 {
                     return Ok(HandleMsgOutput::Ready(quorum_msg));
+                } else if let Vote::Valid(_) = &p.vote {
+                    // ValidationResult from past iteration is found
```

Comment:
Ensure that past iteration is an emergency iteration

Reply:
This should be done in `process_past_events` by checking `msg.iteration` instead of `self.iteration`